src/main/java/org/ccci/util/mail/MailMessageFactory.java

```diff
@@ -92,7 +93,17 @@ public MailMessage createApplicationMessage()
         checkTimeoutProperty();
         return new MailMessage(mailSession, false);
     }
-    
+
+    /**
+     * Creates a {@link MailMessage} suitable for application/business email messages.
+     * These emails might be indirected by {@link MailIndirection}, if configured.
+     */
+    public MailMessage createApplicationMessage(Charset charset)

```

Comment:
Any value calling this here https://github.com/CruGlobal/util/blob/master/src/main/java/org/ccci/util/mail/ErrorEmailer.java#L18?

Reply:
I dont think so as those errors will be in ascii would they not?